Octokit/Clients/UserAdministrationClient.cs

```diff
@@ -1,13 +1,11 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
+﻿using System.Collections.Generic;
+using System.Net;
 using System.Threading.Tasks;
 
 namespace Octokit
 {
     /// <summary>
-    /// A client for GitHub's User Administration API.
+    /// A client for GitHub's User Administration API (GitHub Enterprise)
```

Comment:
Because this client is only used for Enterprise, we should consider adding a check to the constructor that ensures that the `ApiConnection.Connection.BaseUrl` is an Enterprise URL and if not, throw an exception that indicates that this client only works for GH:E. That would require that we create this instance lazily of course.

Perhaps an idea for a separate PR.